jest/src/test/java/io/searchbox/client/http/ContentLengthLimitedInputStreamTest.java

```diff
@@ -0,0 +1,33 @@
+package io.searchbox.client.http;
+
+import io.searchbox.client.http.apache.ContentLengthLimitedInputStream;
+import org.apache.commons.io.IOUtils;
+import org.apache.http.ContentTooLongException;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+
+import static org.junit.Assert.assertEquals;
+
+public class ContentLengthLimitedInputStreamTest {
+
+    @Test
+    public void contentLimitedInputStream() throws IOException {
+        // 57 bytes
+        String initialString = "this should be too long and throw ContentTooLongException";
```

Comment:
This string is actually not too long. Kinda confusing. Not a blocker, but fix it later.

Reply:
Oh right damn I did it right on the limit I defined instead of something clearer, I write really shitty code when I'm at the office <_>